src/extensionApi.ts

```diff
@@ -297,7 +297,7 @@ export class CommandHandler {
             const workflowMap = {};
             for (const item of response1.items) {
                 if (this.isMultilineText(item.content) ) {
-                    await new EditorUtil().showEditor(item.id, item.content);
+                    await new EditorUtil(this.explorer).showEditor(item.id, item.content);
```

Comment:
shouldn't we use ```EditorUtils.getInstance``` (and have the constructor in EditorUtils private) since we're set on a singleton pattern?